ux.symfony.com/assets/controllers/code-highlighter-controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from '@hotwired/stimulus';
+import hljs from 'highlight.js/lib/core';
+import hljs_javascript from 'highlight.js/lib/languages/javascript';
+import hljs_php from 'highlight.js/lib/languages/php';
+// import hljs_twig from 'highlight.js/lib/languages/twig';
```

Comment:
Why add something as a comment if you don't need it?

Reply:
Because i will need it soon..

.. but you're 100% right, as "soon" is not "now", and it could add confusion to anyone reading this code.